libs/src/utils/network.js

```diff
@@ -109,8 +109,39 @@ async function raceRequests (
   return { respone: null, errored }
 }
 
+async function allRequests ({
```

Comment:
nit: can we add a comment explaining this guy?

Reply:
yessir